package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@warp-ds/uno",
```

Comment:
Been wondering about this... is there a reason why the package is named `uno` and not `drive`? 🤔😅

Reply:
Users have no relation to the word `drive`. They're using this package in an UnoCSS context. We don't need to be filling users' heads with "OK I want the Warp Uno preset, was that `@warp-ds/drive` or `@warp-ds/speed` or the other gimmicky package name they went with?" 😆 

`preset-uno` might be an even better name - but it's possible at some point we want to export more than just a preset from this package.

Reply:
...then my next question is: Why is it called `drive` and not `uno` in github? 🤷‍♂️😅😜  I was a bit confused about this since all other repos is mapped 1:1 between github and npm? (Or at least I think they are? 🤔😄)

Reply:
Basically "for the lulz". I agree with your logic, we could have some kind of alias repo to be sure people find the right thing. I'm up for whatever the team feels is best here to aid users understanding our packages and being able to discover things.